src/test/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapperTests.java

```diff
@@ -771,7 +773,7 @@ public void testMethodFieldMapperParseCreateField_validInput_thenDifferentFieldT
         MockedStatic<KNNVectorFieldMapperUtil> utilMockedStatic = Mockito.mockStatic(KNNVectorFieldMapperUtil.class);
         for (VectorDataType dataType : VectorDataType.values()) {
             log.info("Vector Data Type is : {}", dataType);
-            int dimension = dataType == VectorDataType.BINARY ? TEST_DIMENSION * 8 : TEST_DIMENSION;
+            int dimension = dataType == VectorDataType.BINARY ? TEST_DIMENSION * Byte.SIZE : TEST_DIMENSION;
```

Comment:
Byte.SIZE is one I believe

Reply:
No it is actual 8.
![Screenshot 2024-08-16 at 10 37 36 AM](https://github.com/user-attachments/assets/db912b37-aeaa-4a4f-8731-a016da508e15)

Here is the SS.

Reply:
Byte.SIZE provide the bits in the byte. I hope this ans the question